exasol/analytics/query_handler/query/select.py

```diff
@@ -15,6 +17,34 @@ def query_string(self) -> str:
         return self._query_string
 
 
+class WriteQuery(SelectQuery):
+    """
+    A query wrapper for a query that changes data in a database table (e.g. INSERT or
+    UPDATE) or creates the table (e.g. CREATE TABLE). This type of query is auditable.
+    """
+    def __init__(self, query_string: str, affected_table: str, audit: bool = False):
```

Comment:
if we want to do it this genric , I would like to add something like a operation (CREATE TABLE, INSERT TABLE)

Reply:
Yes, sure, why not?